Cookbook/Technical-Documents/OutOffOfficeRequest.md

```diff
@@ -55,5 +66,5 @@ When working from home you should notify in the the iOS team Slack channel when
 
 ### Steps to notify
 
-Notify the iOS Team and your Squad via Slack on the working day before you are working from home. 
-Emergencies might happen that require you to work from home and only notify on the same day. That will be acceptable if there is a plausible reason for it.
+1. Notify the iOS Team and your Squad via Slack on the working day before you are working from home. Emergencies might happen that require you to work from home and only notify on the same day. That will be acceptable if there is a plausible reason for it.
```

Comment:
I'm not a fan of the use of "plausible" here. Not sure if it's because I'm not native or not, but it feel to me like it has a connotation of "we will believe your excuse"... as if it was about lying to justify the WFH reason... Maybe use "an admissible reason" instead?

Reply:
```suggestion
1. Notify the iOS Team and your Squad via Slack on the working day before you are working from home. Emergencies might happen that require you to work from home and only notify on the same day. That will be acceptable if there is a reasonable excuse for it.
```